src/Psalm/Type.php

```diff
@@ -68,14 +68,14 @@ abstract class Type
      */
     public static function parseString(
         string $type_string,
-        ?array $php_version = null,
+        ?int $analysis_php_version_id = null,
```

Comment:
We could accept both forms here, and transform the array to integer to avoid the BC break.